diff --git a/node_modules/vite-plugin-mix/dist/index.js b/node_modules/vite-plugin-mix/dist/index.js
index b0ad03e..377dd58 100644
--- a/node_modules/vite-plugin-mix/dist/index.js
+++ b/node_modules/vite-plugin-mix/dist/index.js
@@ -950,10 +950,6 @@ var index = ({
         await adapter.buildStart(buildOpts);
       }
 
-      const indexHtmlPath = path__default['default'].join(clientOutDir, 'index.html');
-      const indexHtml = fs__default['default'].readFileSync(indexHtmlPath, 'utf8');
-      fs__default['default'].unlinkSync(indexHtmlPath);
-
       await vite.build({
         root,
         resolve: {
@@ -965,7 +961,6 @@ var index = ({
           'import.meta.env.MIX_CLIENT_DIR': JSON.stringify(
             path__default['default'].relative(process.cwd(), clientOutDir),
           ),
-          'import.meta.env.MIX_HTML': JSON.stringify(indexHtml),
         },
         build: {
           outDir: serverOutDir,
diff --git a/node_modules/vite-plugin-mix/dist/runtime/apply-handler-7a6aae3b.js b/node_modules/vite-plugin-mix/dist/runtime/apply-handler-7a6aae3b.js
index 6e6ad88..2d4161c 100644
--- a/node_modules/vite-plugin-mix/dist/runtime/apply-handler-7a6aae3b.js
+++ b/node_modules/vite-plugin-mix/dist/runtime/apply-handler-7a6aae3b.js
@@ -320,18 +320,12 @@ class Polka extends trouter {
 
 var polka = opts => new Polka(opts);
 
-const serveIndex = (req, res) => {
-  res.setHeader('Content-Type', 'text/html');
-  res.end(import.meta.env.MIX_HTML);
-};
-
 const applyHandler = (server) => {
   if (Array.isArray(handler)) {
     handler.forEach((h) => server.use(h));
   } else {
     server.use(handler);
   }
-  server.use(serveIndex);
 };
 
 export { applyHandler as a, polka as p };
